<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TradeFundDialog</class>
 <widget class="QDialog" name="TradeFundDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基金交易</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>从</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="startDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2026</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>到</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="endDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2026</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showButton">
       <property name="text">
        <string>显示</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::AllEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>3</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showButton</sender>
   <signal>clicked()</signal>
   <receiver>TradeFundDialog</receiver>
   <slot>onShowButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1025</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>1251</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>TradeFundDialog</receiver>
   <slot>onSaveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>924</x>
     <y>753</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>968</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>TradeFundDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1012</x>
     <y>750</y>
    </hint>
    <hint type="destinationlabel">
     <x>1193</x>
     <y>871</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>TradeFundDialog</receiver>
   <slot>onTableCellChanged(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>955</x>
     <y>1193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShowButtonClicked()</slot>
  <slot>onSaveButtonClicked()</slot>
  <slot>onTableCellChanged(int,int)</slot>
 </slots>
</ui>
